plugin/src/main/java/net/dzikoysk/funnyguilds/user/User.java

```diff
@@ -28,6 +25,8 @@
     private Option<UserBan> ban = Option.none();
     private final BossBarProvider bossBarProvider;
 
+    private UserProfile profile = UserProfile.NONE;
```

Comment:
Co jeżeli gracza nie ma na serwerze, a my zrobimy np. `user.hasPermission("admin")`?

Reply:
jeśli User jest wczytywany z pluginu to w bazie zostanie setowany profil bukkit i poda rzeczywistą permisje
jeśli to testy to `false` chyba że została dodany testowy profil tak jak tutaj:
![image](https://user-images.githubusercontent.com/49173834/158711107-0dc9ec08-f8b2-4599-b470-5778680b462b.png)

Reply:
To czy de facto istnieje User, który ma UserProfile.NONE? Bo to wygląda jakby jedyne usage dla NONE, to były testy, które mają 2 warainty czyli NONE by default + ta testowa impl

Reply:
Ze względu na ostatnie zmiany to w praktyce nie istnieje na serwerze. Istnieje na testach jednostkowych ewentualnie jakby ktoś z API coś tworzył sobie.

Reply:
Do testów to ma być FakeUserProfile i tyle, proszę w takim razie wywalić tego Missing i zrobić to pole finalne i bedzie gitówa

Reply:
Rozumiem, że wymagane w konstruktorze. To spoko rozwiązanie

Reply:
jednak jest potrzebny missing
![image](https://user-images.githubusercontent.com/49173834/158711860-8c4e3787-69b4-4038-86b0-3a4fd1f5d50e.png)

Reply:
robimy fejk userów tutaj